readthedocs_theme/templates/includes/footer.html

```diff
@@ -30,6 +30,7 @@ <h4 class="ui sub header">Learn more</h4>
               <a class="item" href="/comparisons/github-pages/">Comparison to GitHub Pages</a>
               <a class="item" href="/comparisons/gitbook/">Comparison to GitBook</a>
               <a class="item" href="/comparisons/cloudflare-pages/">Comparison to Cloudflare Pages</a>
+              <a class="item" href="/docs-as-code/">Docs as code</a>
```

Comment:
We're definitely piling more into the footer here. I think it's about time to break these out into links in the main content and a comparisons subpage.

Reply:
Yeah. I added it here because it's the easy one, but we need to find a better pattern. What are you doing in EthicalAds for these pages? Where are you linking them from? Do you have other index pages that link these all?

Reply:
@humitos generally they are linked from inside another page. The "docs as code" page for example could just be linked from copy on our features page talking about this, or something.